CMSgov/mdct-mcr
# Cleanup

## Description

There was a bunch of stuff in here that we did not want to be in here. So we removed it.

### How to test

1. Pull down branch
2. `nvm use`
3.  `./dev local`

Everything should still be up and running. Maybe not looking the greatest, but we'll get to that later.

### Dependencies

No changed deps.

## Code author checklist
- [x] ~~I have added [thorough tests](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)~~

## Reviewer checklist (two different people)
